GliaWidgets/Sources/ViewModel/Call/CallViewModel.swift

```diff
@@ -325,6 +325,17 @@ extension CallViewModel {
         action?(.setLocalVideo(stream.getStreamView()))
         guard call.isVisitorOnHold.value == false else { return }
         stream.playVideo()
+        // For now camera manager is available only when corresponding
+        // local stream is available, that's why camera button presence
```

Comment:
"that's" to "that is" to be consistent with support docs.

```suggestion
        // local stream is available, that is why camera button presence
```